inst/python/gateway.py

```diff
@@ -6,16 +6,23 @@
  
 class GeneratorWrapper():
     def __init__(self, wrapped):
-       object.__setattr__(self, 'inner', wrapped)
+       self._inner = wrapped
+       self._use_list = False
+       self._use_iter = False
 
-    def __getattr__(self, attr):
-        return getattr(self.inner, attr)
-
-    def __setattr__(self, attr, value):
-        setattr(self.inner, attr, value)
-    
     def nextElem(self):
-        return self.__next__()
+      if self._use_list:
+        raise "asList() has been used."
```

Comment:
"Have already enumerated all elements."
(Or just remove this check and let the underlying generator raise an error.)
The latter should only be done if the message is informative.

Reply:
The original error: 
```
Error in pyCall("__R__.namespace[2].nextElem", args = xargs, kwargs = xkwargs) : 
  error occurred while calling '__R__.namespace[2].nextElem'
```